app/Http/Controllers/Controller.php

```diff
@@ -10,4 +10,8 @@
 abstract class Controller extends BaseController
 {
     use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
+
+    public function test_method($key, $defaultValue)
```

Comment:
ERROR: [CamelCaseMethodName] The method test_method is not named in camelCase.
#